src/js/Node.js

```diff
@@ -9,6 +9,44 @@ define(['./appendNodeFactory', './util'], function (appendNodeFactory, util) {
       return obj === Object(obj);
   }
 
+  function _pathIsInReadOnlyPaths(aimaraPath, readOnlyAimaraPaths) {
+    var currentPath, areEquals;
+
+    for (var iPaths = 0, lPaths=readOnlyAimaraPaths.length; iPaths < lPaths; iPaths++) {
+      currentPath = readOnlyAimaraPaths[iPaths];
+      areEquals = false;
+
+      if (aimaraPath.length == currentPath.length) {
+        areEquals = true;
+        for (var iCurrentPath = 0, lCurrentPath=currentPath.length; iCurrentPath < lCurrentPath; iCurrentPath++) {
+          if (aimaraPath[iCurrentPath] != currentPath[iCurrentPath]) { 
+            areEquals = false;   
+            iCurrentPath = lCurrentPath;
```

Comment:
My bad. Initially was breaking both loops at the same time so I needed to set both loop variables to stop them, but then I changed approaches and that wasn't longer needed.